Add reusable write to file component to getting started examples (#839)
ml6team/fondant: docs/guides/implement_custom_components.md

We now have a pipeline that downloads a dataset from the HuggingFace hub, filters the urls by
image type, downloads the images, and filters them by alt text language.

If you want to inspect your final dataset without using the data explorer or use the 
dataset for further tasks, we recommend to write the final dataset to a destination. 

> We offer a write component to perform this task. You can leverage the `write_to_file` component, 

Reviewer: ```suggestion
We offer [write components](../components//hub.md) to perform this task, for instance the `write_to_file` component, 
```
